src/modules/toolbox/components/shareButton/ShareDialogContent.js

```diff
@@ -128,9 +161,18 @@ export class ShareDialogContent extends MvuElement {
 		}
 	}
 
-	set shareurls(value) {
-		this._shareUrls = value;
-		this.render();
+	/**
+	 * sets the urls for the dialog to share
+	 * @param {String|FileSaveUrl} value the shared value; wether a plain string (is interpreted as a valid wellformed url) or a FileSaveUrl-Object
```

Comment:
Shouldn't it be `FileSaveResult` instead of `FileSaveUrl`?
https://github.com/ldbv-by/bav4/blob/d9c17d5da2fa610f610cf2009af4eb7361abac31/src/services/FileStorageService.js#L64